packages/bun-types/test.d.ts

```diff
@@ -13,20 +13,14 @@
  * $ bun test <filename>
  * ```
  */
-
 declare module "bun:test" {
-  type AnyFunction = (...args: any) => any;
   /**
    * -- Mocks --
    */
-  export interface Mock<T extends AnyFunction>
-    extends JestMock.MockInstance<T> {
-    (...args: Parameters<T>): ReturnType<T>;
-  }
-  type _Mock<T extends AnyFunction> = Mock<T>;
+  export type Mock<T extends (...args: any[]) => any> = JestMock.Mock<T>;
 
   export const mock: {
-    <T extends AnyFunction>(Function: T): Mock<T>;
+    <T extends (...args: any[]) => any>(Function: T): Mock<T>;
```

Comment:
This comment is in regards to all usage of `(...args: any[])`. Can we use `(...args: unknown[])` instead? Especially if we are not returning the types anywhere.

Reply:
We cannot in the node apis and globals, since `bun:test` is a bun api we could change it, but, it might affect some specific users already using it.

Reply:
Makes sense, if that's the case we can just keep `any` as-is.